src/test/java/com/openlattice/shuttle/transforms/TransformTest.java

```diff
@@ -328,20 +328,96 @@ public void testReplaceTransform() {
 
     }
 
-    @Test(timeout=3000)
+    @Test(timeout=4000)
     public void testGeocoderTransform() {
         String expectedStreet = "Scott Street";
         Object geocoderTest1 = new GeocoderTransform( "road", Optional.empty() )
                 .applyValue( getTestRow().get( "Address" ) );
+
         String expectedNo = "560";
         Object geocoderTest2 = new GeocoderTransform( "house_number", Optional.empty() )
                 .applyValue( getTestRow().get( "Address" ) );
+
         String expectedType = "house";
         Object geocoderTest3 = new GeocoderTransform( "type", Optional.empty() )
                 .applyValue( getTestRow().get( "Address" ) );
+
+        String expectedGeoPoint = "37.7748313877551,-122.435947714286";
+        Object geocoderTest4 = new GeocoderTransform( "geographypoint", Optional.empty() )
+                .applyValue( getTestRow().get( "Address" ) );
+
         Assert.assertEquals( expectedStreet, geocoderTest1 );
         Assert.assertEquals( expectedNo, geocoderTest2 );
         Assert.assertEquals( expectedType, geocoderTest3 );
+        Assert.assertEquals( expectedGeoPoint, geocoderTest4 );
+    }
+
+
+    @Test
+    public void testArithmeticTransform() {
+        double expectedSum = 36.23452 + 30.34573;
+        double expectedDiff = 36.23452 - 30.34573;
+        double expectedProd = 36.23452 * 30.34573;
+        double expectedQuo = 36.23452 / 30.34573;
+
```

Comment:
Would prefer an additional test that verifies behavior when an invalid operator is specified